torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -96,6 +96,37 @@ XlaOpVector SizeAdd::Lower(LoweringContext* loctx) const {
   return ReturnOp((input1 + input2), loctx);
 }
 
+SizeSub::SizeSub(torch::lazy::Value a, torch::lazy::Value b)
+    : XlaNode(torch::lazy::OpKind{c10::Symbol::fromQualString(
+                  "aten::sub")},  // TODO: should it be something like
```

Comment:
I feel the OpKind here can be more specific such as "aten::size_sub", as opposed to the general ones such as "aten::sub". wdyt? @miladm @JackCaoG @alanwaketan

Reply:
c10::Symbol is something defined in upstream. If you can find the corresponding enum in upstream, I guess it's better. Yea, we don't store "aten::sub" as a string. It's an enum.

Reply:
I guess the enums are defined in pytorch/aten/src/ATen/core/interned_strings.h? `size_sub` is not one of the enums but I could still compile locally and run the test.